### PR TITLE
Improve loading times for slow connections

### PR DESCRIPTION
**Pull request by codewiththeflowxyz**

### CODE REVIEW

File: app/(main)/components/LandingContent.tsx [hunk 1]
```diff
@@ -10,15 +10,30 @@ import { FAQ } from './FAQ';
 import { Footer } from '@/app/components/Footer';
 import { FetchedData } from '../fetchData';
 import { TrianglesSeparator } from '@/app/components/TrianglesSeparator';
+import { useAppContext } from '@/app/context/AppContext';
 
 export function LandingContent({ data }: { data: FetchedData }) {
   const childrenRef = useRef<HTMLDivElement>(null);
+  const { landingLoadingTime, setLandingLoadingTime, setIsSlowNetwork, isSlowNetwork } = useAppContext();
+  console.log('🚀 ~ LandingContent ~ landingLoadingTime:', landingLoadingTime);
+  console.log('🚀 ~ LandingContent ~ isSlowNetwork:', isSlowNetwork);
```
Comment (bitsforpeace) [inline app/(main)/components/LandingContent.tsx:19]: ✂️

**Review by bitsforpeace — COMMENTED** [review]
Main page loads well for me on 3G and slow 4G. Should we add similar logic on the Features page?

**Review by bitsforpeace — APPROVED** [review]

**Review by iceanddust — CHANGES_REQUESTED** [review]
Nice work, it loads much faster and consistently now on slow connections!

I think I found a bug, though, when I try to load the home page in slow 4G with cache disabled, the tokens section disappeared. Even after the fully loading the site, the section remains hidden. This is not the case for when you reload the site at full speed

**Review by iceanddust — APPROVED** [review]
Approving as the change is intended

**Review by generatorpoint — APPROVED** [review]
Nice, HUGE improvement from before. I'm still noticing a very short lag on first load, but I don't notice it on the features page, even when loading the page in a private window for the first time

**Pull request merged**